beacon-chain/sync/subscriber_beacon_blocks_chunks.go

```diff
@@ -17,11 +17,14 @@ import (
 	"github.com/prysmaticlabs/prysm/v5/consensus-types/interfaces"
 	"github.com/prysmaticlabs/prysm/v5/monitoring/tracing"
 	"github.com/prysmaticlabs/prysm/v5/monitoring/tracing/trace"
+	ethpb "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
 	"github.com/prysmaticlabs/prysm/v5/time/slots"
 	"github.com/sirupsen/logrus"
 	"google.golang.org/protobuf/proto"
 )
 
+const meshSize = 40 // number of peers to send chunks
```

Comment:
This is hardcoded and repeated, should be moved to a config or to the p2p package.